static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.spec.ts

```diff
@@ -102,7 +102,6 @@ describe('initialise', () => {
 			s2sConfig: {
 				adapter: 'prebidServer',
 				adapterOptions: {},
-				enabled: false,
```

Comment:
👍 

We upgrade this config nearly every time we make a major update.

Reply:
That is reassuring, thanks